kie-server-parent/kie-server-remote/kie-server-client/src/main/java/org/kie/server/client/impl/JobServicesClientImpl.java

```diff
@@ -179,6 +209,38 @@ public void requeueRequest(long requestId) {
     }
 
     @Override
+    public List<RequestInfoInstance> getRequestsByBusinessKey(String businessKey, List<String> statuses, Integer page, Integer pageSize) {
+        RequestInfoInstanceList list = null;
+        if( config.isRest() ) {
```

Comment:
Whenever I see this construction in Kie-Server Client, I wonder if it would be possible to refactor the code to have two client implementations and a factory deciding which one of them to provide based on `config.isRest()`. I am not saying it should be done now for this PR, just making a suggestion we can discuss further.